src/descriptor/mod.rs

```diff
@@ -68,6 +75,258 @@ pub enum Descriptor<Pk: MiniscriptKey> {
     ShWsh(Miniscript<Pk, Segwitv0>),
 }
 
+#[derive(Debug, Eq, PartialEq, Clone, Ord, PartialOrd, Hash)]
+pub enum DescriptorPublicKey {
+    ShortPub(DescriptorShortPub),
```

Comment:
I'd rather call it `DescriptorSinglePub` to avoid confusion with the compressed/uncompressed dichotomy.

Reply:
Yes, that's a better name, thanks.